mixins/localize/localize-mixin.js

```diff
@@ -63,6 +61,8 @@ export const _LocalizeMixinBase = dedupeMixin(superclass => class LocalizeMixinB
 		this.requestUpdate('localize');
 	}
 
+	#updatedProperties = new Map();
```

Comment:
I do find defining the private properties down here a little weird. But that's the order we always had for them.